twilight-gateway/src/event.rs

```diff
@@ -423,40 +426,18 @@ impl From<EventType> for EventTypeFlags {
     }
 }
 
-impl<'a> TryFrom<(u8, Option<&'a str>)> for EventTypeFlags {
-    type Error = (u8, Option<&'a str>);
-
-    fn try_from((op, event_type): (u8, Option<&'a str>)) -> Result<Self, Self::Error> {
-        match (op, event_type) {
-            (1, _) => Ok(EventTypeFlags::GATEWAY_HEARTBEAT),
-            (7, _) => Ok(EventTypeFlags::GATEWAY_RECONNECT),
-            (9, _) => Ok(EventTypeFlags::GATEWAY_INVALIDATE_SESSION),
-            (10, _) => Ok(EventTypeFlags::GATEWAY_HELLO),
-            (11, _) => Ok(EventTypeFlags::GATEWAY_HEARTBEAT_ACK),
-            (_, Some(event_type)) => {
-                let flag = EventType::try_from(event_type).map_err(|kind| (op, Some(kind)))?;
-
-                Ok(Self::from(flag))
-            }
-            (_, None) => Err((op, event_type)),
-        }
-    }
-}
-
 impl Default for EventTypeFlags {
```

Comment:
`Default` should only be implemented for types with an obvious default. I don't think that's the case here (could be assumed to be an empty set just as well as a full set of flags), and it should therefore be removed.